public/js/components/Charts/AreaChartWrap.js

```diff
@@ -15,20 +16,28 @@ const AreaChartWrap = ({ title, series, xLabel, yLabel, isUpdating, scale, stack
     
     return (
         <div className={getClassName()}>
+            <h3>Tool of Origin: <span style={{color: '#33a22b'}}>InCopy</span> vs <span style={{color: '#ffbc01'}}>Composer</span></h3>
```

Comment:
Does it make more sense for the style to be in classes in css? So we can use a value in `_variables.scss` and if the graph colours change the title changes as well